tests/timeseries/_battery_pool/test_battery_pool_control_methods.py

```diff
@@ -213,6 +213,8 @@ async def test_case_1(
             await bounds_rx.receive(), power=1000.0, lower=-4000.0, upper=4000.0
         )
 
+        await asyncio.sleep(0.0)
```

Comment:
Please document why all these new sleeps are needed.

Reply:
This was just a failed attempt to fix tests. I prefer to validate the code-changes rather than jumping in fixing tests and docs. This is the reason I keep it as a draft, to show and to discuss the changes. Sorry, I should have added a note/comment for this

Reply:
Updated